stdlib/public/runtime/Errors.cpp

```diff
@@ -22,11 +22,124 @@
 #include <pthread.h>
 #include <stdarg.h>
 #include "swift/Runtime/Debug.h"
+#include "swift/Basic/Demangle.h"
+#include <cxxabi.h>
+#include <execinfo.h>
 
 #ifdef __APPLE__
 #include <asl.h>
 #endif
 
+typedef enum FatalErrorFlags : uint32_t {
```

Comment:
I don't think you need 'typedef' in C++, just an enum decl should be sufficient.